partition/roles/sonic/templates/frr.conf.j2

```diff
@@ -136,7 +145,9 @@ router bgp {{ sonic_asn }} vrf {{ i.vrf }}
   {{ a }}
     {% endfor %}
     {% endif %}
+    {% if i.keep_private_as is not defined or i.keep_private_as is false %}
```

Comment:
I think this changes configuration for existing deployments because it introduces a new variable and removes this line when it was not defined?

Reply:
It removes the line if is defined and true. Looks safe for me.

Reply:
It renders the `remove-private-AS` statement if the `keep_private_as variable` is not defined or false; existing deployments do not have the variable defined so it should not change anything.

Also it does not break the `exit` test case. :)

Reply:
That's true, I  misread the condition. Thanks for clarification.